src/App.js

```diff
@@ -58,14 +98,17 @@ const App = () => {
     <div className="App">
       <header className="App-header">
         <h1>React Tic Tac Toe</h1>
-        <h2>The winner is ... -- Fill in for wave 3 </h2>
+        <h2>Winner is: {checkForWinner()} </h2>
+        <h2>{`Current Player ${player}`}</h2>
```

Comment:
Once a winner has been found, we don't want to display the current player info anymore. So which of these we want to display is really governed by whether we've found a winner yet. This is another argument in favor of adding some winner state (there _are_ ways to avoid this, but using the state is probably the most direct). Once we have that `winner` state (as shown above), we could rewrite these lines to use that state as something like:

```js
        <h2>{ winner ? `Winner is ${winner}` : `Current Player ${player}` }</h2>
```

Notice that I removed the `:` from the winner display. This is to make the tests happy. The winner check doesn't expect a `:` in the display string. Of course, if you want to include a `:`, the test could be updated to match that.